go/vt/vttablet/tabletmanager/tm_init_test.go

```diff
@@ -422,9 +422,10 @@ func TestStartCheckMysql(t *testing.T) {
 	assert.Equal(t, "foo", ti.MysqlHostname)
 }
 
+// TestStartFindMysqlPort tests the functionality of findMySQLPort on tablet startup
```

Comment:
have you made sure this test passes continuously locally after running it a lot of times?

Reply:
Yes, it does 👍